components/table-accordion-group/README.md

```diff
@@ -0,0 +1,164 @@
+TableAccordionGroup
+===================
+
+### Import
+```js
+  import TableAccordionGroup from '@govuk-frederic/table-accordion-group';
+```
+<!-- STORY -->
+
+### Usage
+
+Simple
+```jsx
+<TableAccordionGroup title="Title" expanded="expanded">Children</TableAccordionGroup>
+```
+
+Open
```

Comment:
It would be nice if these examples were things that people could copy and paste for use in their project - I feel the examples on this page either don't work as expected or have a dependency that is not explained.